### DISCUSSION TITLE
Connected to OPC HDA Server, but all requests fail

### DISCUSSION THREAD

**Discussion by gilsalvans** [duduyoyo/WebSocket4OPC#13] [Q&A] [answered]
Hello,
I installed following the readme this service and everything seems to look fine on IIS side. I can access the application and it even says "Connected". However, when I make a request (? - help request, 'readRaw', ....) with the right Tag from our OPC Server, it always fails. The error is following:  "detected but module is out of scope. Looks like a lapsed request"

I attach here the Events from the Windows Event Log, so you can see that the connection seems to be ok, but the readRaw command fails. 
[Websocket_Events.csv](https://github.com/user-attachments/files/15951667/Websocket_Events.csv)

Best
Gil

**Comment by duduyoyo** [comment, marked as answer]
Your Windows event log file is helpful in spotting a bug in my code quickly. Thanks!

Please download from link as below and unzip it. Replace the same file at C:\Windows\System32\inetsrv. Give it a shot and let me know. Cheers!

https://drive.google.com/file/d/1zRN_abUhy6JLxe3O78X1ygIx-AuhMMiI/view?usp=sharing

**Reply by gilsalvans** [reply]
Thank you for fixing the bug! The file replacement did the trick.
Best :)
